brainscore_vision/model_helpers/activations/pytorch.py

```diff
@@ -137,16 +137,35 @@ def preprocess_images(images, image_size, **kwargs):
 
 def torchvision_preprocess_input(image_size, **kwargs):
     from torchvision import transforms
-    return transforms.Compose([
-        transforms.Resize((image_size, image_size)),
-        torchvision_preprocess(**kwargs),
-    ])
-
-
-def torchvision_preprocess(normalize_mean=(0.485, 0.456, 0.406), normalize_std=(0.229, 0.224, 0.225)):
+    preprocess_type = kwargs.get('preprocess_type', 'imagenet')
+    if preprocess_type.lower() == 'inception':
+        # inception-style resize
+        resize_size = int(image_size * 256 / 224)
+        return transforms.Compose([
+            transforms.Resize(resize_size),
+            transforms.CenterCrop(image_size),
+            torchvision_preprocess(preprocess_type='inception')
+        ])
+    else:
+        return transforms.Compose([
+            transforms.Resize((image_size, image_size)),
+            torchvision_preprocess(**kwargs),
+        ])
```

Comment:
```suggestion
    preprocess_type = kwargs.get('preprocess_type', 'imagenet').lower()
    if preprocess_type == 'imagenet':
        return transforms.Compose([
            transforms.Resize((image_size, image_size)),
            torchvision_preprocess(**kwargs),
        ])
    elif preprocess_type == 'inception':  # inception-style resize
        resize_size = int(image_size * 256 / 224)
        return transforms.Compose([
            transforms.Resize(resize_size),
            transforms.CenterCrop(image_size),
            torchvision_preprocess(preprocess_type='inception')
        ])
    else:
        raise ValueError(f"Unknown preprocess_type '{preprocess_type}'")
```

Reply:
note that I'm passing `**kwargs` into `torchvision_preprocess(**kwargs)` again which was deleted before. I think this is correct (`preprocess_type` should already be part of `kwargs` and otherwise we're losing other arguments) but please double-check.